packages/babel-generator/src/index.js

```diff
@@ -58,7 +58,6 @@ function normalizeOptions(code, opts, tokens): Format {
     compact: opts.compact,
     minified: opts.minified,
     concise: opts.concise,
-    quotes: opts.quotes || findCommonStringDelimiter(code, tokens),
```

Comment:
@4rlekin Can we just remove the top level quotes option? Basically I think we can just do `quotes: findCommonStringDelimiter(code, tokens),` so that it still runs findCommonStringDelimiter. Basically we just need to not allow the user to specify the quotes and rather we infer it.